src/controllers/login.js

```diff
@@ -1,3 +1,40 @@
+const bcrypt = require('bcryptjs');
+const getData = require('../../src/database/queries/getData');
+const { validationRegex } = require('../../public/js/helpers/signup');
+
+exports.get = (request, response) => {
+  response.render('login', {
+    js: ['/helpers/collectData', 'login'],
+    css: ['login'],
+    layout: 'login_signup',
+    title: 'Login',
+  });
+};
+exports.post = (request, response, next) => {
+  if (validationRegex.firstStepValidationRegex[3].test(request.body.email.trim())) {
```

Comment:
If `firstStepValidationRegex` and `thirdStepValidationRegex` were objects instead of arrays, this code would be a little easier to read.  Then it could look like:
`if (validationRegex.firstStepValidationRegex.email.test(request.body.email.trim())) {`

Also, then you would never have the problem where you add a new item in the middle of the firstStepValidationRegex array, and then you need to update all of these indices from 3 to 4, 0 to 1, because of the array being shifted.